src/pages/index.js

```diff
@@ -26,7 +47,40 @@ const Home = ({ data, location }) => {
       />
 
       <StickyFooter location={location}>
-        <Hero setModalOpen={setModalOpen} siteMetadata={data.site.siteMetadata} />
+        <ResponsiveSpacer>
+          <Hero setModalOpen={setModalOpen} siteMetadata={data.site.siteMetadata} />
+        </ResponsiveSpacer>
+
+        <ResponsiveSpacer>
+          <InterstitialTitle fontSize="3em">
+            Do away with tribal knowledge
```

Comment:
Just an idea maybe "invert" the suggestion suggest that they can put their tribal knowledge into backstage and _speed up_ development. On the other hand I suppose you're trying to get them to think of the problem we're solving...